homeassistant/components/media_player/mpd.py

```diff
@@ -319,7 +319,10 @@ def play_media(self, media_type, media_id, **kwargs):
     @property
     def shuffle(self):
         """Boolean if shuffle is enabled."""
-        return bool(self._status['random'])
+        if self._status['random'] == '1':
+            return 'true'
+        else:
+            return 'false'
```

Comment:
This is not a boolean. That wouldn't have worked.

Reply:
It does seem to work as it's what I've been running, but yours is a much better solution.  I'll change it and test it out.